sdk/storage/storage-blob-change-feed/samples/typscript/basic.ts

```diff
@@ -0,0 +1,48 @@
+import { BlobServiceClient, StorageSharedKeyCredential } from "@azure/storage-blob";
+// import { BlobChangeFeedClient } from "@azure/storage-blob-change-feed";
+import { BlobChangeFeedClient } from "../../src";
+
+// Load the .env file if it exists
+import * as dotenv from "dotenv";
+console.log(dotenv.config());
+
+import { setLogLevel } from "@azure/logger";
+setLogLevel("info");
+
+export async function main() {
+  // Enter your storage account name and shared key
+  const account = process.env.ACCOUNT_NAME || "";
+  const accountKey = process.env.ACCOUNT_KEY || "";
+
+  // Use StorageSharedKeyCredential with storage account and account key
+  // StorageSharedKeyCredential is only available in Node.js runtime, not in browsers
+  const sharedKeyCredential = new StorageSharedKeyCredential(account, accountKey);
+  const blobServiceClient = new BlobServiceClient(
+    // When using AnonymousCredential, following url should include a valid SAS or support public access
+    `https://${account}.blob.core.windows.net`,
+    sharedKeyCredential
+  );
+
+
+  const containerClient = blobServiceClient.getContainerClient("$blobchangefeed");
+  console.log("List container.")
```

Comment:
might better to elaborate as "List $blobchangefeed container". BTW, what's the E2E scenario for user to list this container for?

Reply:
Will come to tidy up samples later.

Reply:
list container part removed. Only for test.